app/models/product.rb

```diff
@@ -10,7 +11,7 @@ class Product < ApplicationRecord
   accepts_nested_attributes_for :product_prices, allow_destroy: true
 
   def requires_age
-    %w[beer craft_beer distilled wine tobacco].include? category
+    %w[beer craft_beer distilled whiskey wine tobacco].include? category
```

Comment:
is low_alcohol_beer also age-restricted or can anyone drink it? (Not sure on Dutch law on this)

Reply:
Like is it 2.0% Radler and such (probs age restricted?) or 0.0% Radler (is this age restricted?) and such, or something different?

Reply:
It's a category for Lux, we won't use it in Flux. But I asked them and it's beer below 0.5% alcohol, which is not age restricted.

Reply:
:+1: